docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -59,10 +57,9 @@ When does my data become real?
 
```

Comment:
A very good point, the start of this little section is a bit abrupt otherwise isn't it...